.github/workflows/release-backwards-compatiblity.yml

```diff
@@ -0,0 +1,83 @@
+name: Backwards Compatibility Test
+
+on:
+  pull_request:
+    branches:
+      - master
+
+env:
+  SELENIUM_GRID_PROVIDER: saucelabs
+  CLOUD_WATCH_METRIC: false
+  TEST_TYPE: Github-Action
+  SAUCE_USERNAME: ${{secrets.SAUCE_USERNAME}}
+  SAUCE_ACCESS_KEY: ${{secrets.SAUCE_ACCESS_KEY}}
+  MESSAGING_USER_ARN: ${{secrets.MESSAGING_USER_ARN}}
+
+jobs:
+  release-integ-test:
+    name: Run Backwards Compatible Integration Tests with Release Tarball
+    runs-on: ubuntu-latest
+    if: ${{ github.head_ref == 'release' }}
+
+    steps:
+      - name: Setup Node.js - 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Get Current Package version
+        id: package-version
+        uses: martinbeentjes/npm-get-version-action@69d4697a4cbb77459281496ef908153917e63cb2
```

Comment:
Checking if this should be `martinbeentjes/npm-get-version-action@master` based on the action [here](https://github.com/marketplace/actions/get-current-package-version).

Reply:
We want to use the commit hash - this is for security reasons and is best practice. https://dev.to/mheap/improve-your-github-actions-security-1im7

Reply:
Any advantages using this action over adding a script ourselves to get the latest published npm version like we have for getting the previous version [here](https://github.com/aws/amazon-chime-sdk-js/blob/master/.github/script/get-prev-version)?

Reply:
Main benefit is that we don't need to create the script ourselves.